deploy.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# To publish a build, use the Makefile
+# make publish
+# which will call out to this script
+set -e
+
+# Use maven-help-plugin to get the current project.version
+VERSION=`mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -v '\['`
+
+echo "Publishing $VERSION"
+
+# Build and deploy to Bintray
+mvn -s .circleci.settings.xml -Dmaven.test.skip=true deploy -pl .,lightstep-tracer-jre,lightstep-tracer-jre-bundle,shadow
```

Comment:
Wrong components ;) 

We have to use `mvn -s .circleci.settings.xml -Dmaven.test.skip=true deploy`

(I don't think we need to manually select what components we want to deploy, as we want to deploy all our 3 artifacts)

Reply:
done